### PR TITLE
Karma browserselection

### PR DESCRIPTION
**Pull request by c0defre4k**
Add an option to choose which browser should be used for Karma unit tests

### CODE REVIEW

**Comment by c0defre4k** [comment]
I've removed the ModuleConcatentionPlugin from development config as its of course not needed there.
I also updated webpack to version 3. Yes, the title of the PR doesn't indicate that. Should I open another PR or just update the title of this PR?

File: template/package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "css-loader": "^0.28.0",
     "cssnano": "^3.10.0",
     {{#lint}}
-    "eslint": "^3.19.0",
+    "eslint": "^4.4.0",
```
Comment (Aladdin-ADD) [inline template/package.json]: eslint just release a patch v4.4.1

Comment (c0defre4k) [inline template/package.json]: Thats right, but since the version string is **^**4.4.0 is should always install the newest available version < 5.0.0.

Comment (Aladdin-ADD) [inline template/package.json]: there is a small diffence that when has installed eslint@4.4.0.

**Comment by MaxMilton** [comment]
@c0defre4k webpack 3 support is going to be a significant change and definitely should be handled in a separate PR. There's an issue with some background info: https://github.com/vuejs-templates/webpack/issues/751

**Comment by c0defre4k** [comment]
@MaxMilton I see - thanks. I've updated the PR to focus on Karma browserselection

**Comment by LinusBorg** [comment]
We will deprecate the karma setup in the forseeable future in favor of jest.

For that reason, I don't think it's  very useful to extend the questionaire with further options, so I have to close this PR.

Thanks for the efforts nontheless!

**Pull request closed without merging**